Transplant code from F4 to F1

582864732 opened the pull request.

582864732 committed:
Transplant code from F4 to F1

qiayuanl reviewed (changes requested):
整个项目名称叫 imu2can 就行了 没必要改成 imu2ccan_F1, 其他没问题了

582864732 committed:
Rename imu2can_F1 to imu2can

582864732 commented:
把imu2can_F1改名成了imu2can

582864732 committed:
Add CAN receive for triggering camera

582864732 committed:
Update the trigger byte(can_data[6])
bit 1 indicates whether to trigger,
bit 2 indicates whether the trigger is enabled

qiayuanliao commented:
Commit的信息首字母大写

qiayuanliao commented:
@582864732

qiayuanliao commented:
@582864732

qiayuanliao commented:
@582864732
